scalableminds/webknossos-libs
# Enable metadata access for remote datasets.

### Description:
- Implements getters and setters for the new metadata field in remote datasets and folders.

### Issues:
- fixes #1148

### Todos:
Make sure to delete unnecessary points or to check all before merging:
 - [x] Considered adding this to the Examples

 - [x] blocked by https://github.com/scalableminds/webknossos/pull/7886
 
 ### How to test:
 Just run the example in `accessing_metadata.py` and enter your webknossos token when asked. After execution, the dataset `l4_sample` and the folder `Datasets/Demo-Datasets` should have an entry with metadata.
